sonarsource-public.key

```diff
@@ -0,0 +1,64 @@
+-----BEGIN PGP PUBLIC KEY BLOCK-----
```

Comment:
This file could be downloaded from a trusted source at build time, but since it's static (and small) we can also copy it here.